src/components/Notifications.vue

```diff
@@ -27,24 +50,51 @@ import { mapGetters, mapActions } from 'vuex'
 
 export default {
   computed: {
-    ...mapGetters(['activeNotifications'])
+    ...mapGetters(['activeNotifications', 'configuration'])
   },
   methods: {
-    ...mapActions(['deleteNotification']),
-    sortedActions (actions) {
-      actions.sort((x, y) => x.primary === y.primary ? 0 : x.primary ? -1 : 1)
-    },
-    executeRequest (app, link, type) {
+    ...mapActions(['deleteNotification', 'showMessage']),
+    ...mapActions('Files', ['loadFolder']),
+
+    executeRequest (app, link, type, notificationId) {
       this.$client.requests.ocs({
         service: 'apps/' + app,
-        action: link,
+        action: link.substr(link.lastIndexOf('api')),
         method: type
+      }).then(res => {
+        this.deleteNotification({
+          client: this.$client,
+          notification: notificationId
+        })
+        res.json().then(json => {
+          json.ocs.data.map(item => {
+            const path = item.path.substr(0, item.path.lastIndexOf('/') + 1)
+            const absolutePath = this.$route.params.item ? this.$route.params.item : '/'
+            if (path === absolutePath) this.reloadFilesList(path)
+          })
+        })
+      })
+    },
+    reloadFilesList (path) {
+      this.loadFolder({
+        client: this.$client,
+        absolutePath: path,
+        $gettext: this.$gettext,
+        routeName: this.$route.name
+      }).catch((error) => {
+        this.showMessage({
+          title: this.$gettext('Loading folder failed'),
```

Comment:
Please copy texts to make sure we have them consistent and reduce the amount of translations - THX

Reply:
Fixed